modules/rest-api/pages/rest-fts-partition-file-transfer.adoc

```diff
@@ -0,0 +1,79 @@
+= Rebalance with File Transfer
+:description: The Search-Service REST API allows new partitions to be created by file transfer.
```

Comment:
The Search-Service REST API helps to configure the file transfer based rebalance.  ?

Reply:
I've put: The Search-Service REST API configures rebalance based on file transfer.